azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/parser/V2ConfigurationParser.java

```diff
@@ -51,7 +50,7 @@ protected OperatingSystemEnum getOs() throws MojoExecutionException {
     @Override
     protected Region getRegion() throws MojoExecutionException {
         final String region = mojo.getRegion();
-        if (!Arrays.asList(Region.values()).contains(region)) {
+        if (Region.findByLabelOrName(region) == null) {
```

Comment:
What will happen if region is not config in pom here? will it also throw region not support Exception?

Reply:
My fault. That case should be taken care of.